Source/Lib/System/EbEncHandle.c

```diff
@@ -2172,11 +2172,19 @@ static EB_ERRORTYPE VerifySettings(\
         printf( "Error Instance %u: Base Layer Switch Mode 1 only when Prediction Structure is Random Access\n", channelNumber + 1);
         return_error = EB_ErrorBadParameter;
     }
-
-	if (sequenceControlSetPtr->maxInputLumaWidth % 8) {
-        printf("Error Instance %u: Only multiple of 8 width is supported in this release \n",channelNumber+1);
+    if ((sequenceControlSetPtr->maxInputLumaWidth % 8) && config->compressedTenBitFormat == 1) {
```

Comment:
Please clean up similar logic several lines below:
if (sequenceControlSetPtr->maxInputLumaWidth % 8 != 0 && config->encoderBitDepth == 10)
 if (sequenceControlSetPtr->maxInputLumaHeight % 8 != 0 && config->encoderBitDepth == 10) {

Reply:
The compressed 10-bit format can not be a non-multiple of 8 for now.

Reply:
compressedTenBitFormat and encoderBitDepth are not the same and compressedTenBitFormat requires a multiple of 8 by design